frontend/src/app.js

```diff
@@ -132,6 +133,11 @@ const App = () => {
                 path="/Q&A"
                 render={() => <Ques theme={theme} />}
               />
+              <Route
```

Comment:
why do we need a separate route for answers?? each question should have a list of answers right?

Reply:
I think it's more  Efficient for  Data Handling

Reply:
directing to a new page for answers doesn't make sense, either make it collapsible(like FAQ page) or open a pop-up and show the question and all its answers,